code/modules/mob/hear_say.dm

```diff
@@ -95,6 +95,12 @@
 		if(client.prefs.toggles & PREFTOGGLE_CHAT_GHOSTEARS && (speaker in view(src)))
 			message = "<b>[message]</b>"
 
+	// Check if the language used is innate
+	for(var/datum/multilingual_say_piece/SP in message_pieces)
+		if(SP.speaking && SP.speaking.flags & INNATE)
+			custom_emote(EMOTE_VISIBLE, message_clean, TRUE)
```

Comment:
If they're emoting into the microphone, it'd make sense that it's an audible emote!
```suggestion
			custom_emote(EMOTE_AUDIBLE, message_clean, TRUE)
```

Reply:
Good point- makes sense to me!